Assets/Scripts/Player/PlayerHealth.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System.Collections;
+using UnityEngine;
+
+public class PlayerHealth : MonoBehaviour
+{
+    public int healthPoints;
```

Comment:
Protect this with a getter and setter property so that you can ensure that it cannot be set above the player's max health. See [c# docs](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/properties)